### PR TITLE
Feature/refactoring

### PR DESCRIPTION
**Pull request by stancdam**
stage1 refactoring

### CODE REVIEW

File: DataReader/Controller/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+import UIKit
+
+class ViewController: UIViewController, UITableViewDataSource {
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    // TODO: textObjects should be private
+    var textObjects = [String]()
+    private let reuseCellId = "DynamicCell", cellNibName = "DynamicCell"
```
Comment (tomasz-czyzak) [inline DataReader/Controller/ViewController.swift:19]: duplicate variable? with the same value?

File: DataReader/Controller/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    // TODO: delete timer if DataChange feature not needed
+    var timer: Timer?
+    
+    convenience init(dataModel: [String]) {
```
Comment (tomasz-czyzak) [inline DataReader/Controller/ViewController.swift:23]: not needed VC should be created by story board

File: DataReader/AppDelegate.swift [hunk 3]
```diff
@@ -14,7 +14,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-       return true
+        window = UIWindow(frame: UIScreen.main.bounds)
```
Comment (tomasz-czyzak) [inline DataReader/AppDelegate.swift:17]: I know that you wanted to try xib approach but I prefer "less code less bug" aproach

File: DataReader/Service/APIService.swift [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+//  APIService.swift
+//  DataReader
+//
+//  Created by Damian Stanczyk on 02.11.2017.
+//  Copyright © 2017 haze. All rights reserved.
+//
+
+import Foundation
+
+protocol APIService {
```
Comment (tomasz-czyzak) [inline DataReader/Service/APIService.swift:11]: empty class? remove

**Review by tomasz-czyzak — CHANGES_REQUESTED** [review]

File: DataReader/Controller/ViewController.swift (hunk 1 above)
Comment (stancdam) [reply DataReader/Controller/ViewController.swift:19]: Of course its the same value in this case, however one is for xib name and next for cell identifier which may differ and thats the reason why I have two variables

Comment (tomasz-czyzak) [reply DataReader/Controller/ViewController.swift:19]: but they aren't ;)

Comment (stancdam) [reply DataReader/Controller/ViewController.swift:19]: haha right

**Pull request closed without merging**